src/CGIHandler.cpp

```diff
@@ -97,6 +98,17 @@ HTTPResponse CGIHandler::CGIStringToResponse(const std::string &cgiOutput)
 	return response;
 }
 
+void CGIHandler::closeAllSocketFDs()
+{
+	for (std::vector<struct pollfd>::iterator it = _FDsRef->begin(); it != _FDsRef->end(); ++it)
+	{
+		if (it->fd != _pollFd->fd)
```

Comment:
This means that you are not closing the one used by the parent process. Why? We discussed this already I guess ...

Reply:
Done